app/core/api/serializers.py

```diff
@@ -296,3 +297,21 @@ class Meta:
             "created_at",
             "updated_at",
         )
+
+
+class AffiliationSerializer(serializers.ModelSerializer):
+    """
+    Used to retrieve Affiliation
+    """
+
+    class Meta:
+        model = Affiliation
+        fields = (
+            "uuid",
+            "affiliate",
+            "project",
+            "ended_at",
```

Comment:
I think we need to show the "created_at" for this to make sense. Right now, it only shows "ended_at" which looks weird without the begin datetime.
![image](https://github.com/hackforla/peopledepot/assets/1160105/38e43107-5f62-4fdc-b2fc-f437a6888bcf)